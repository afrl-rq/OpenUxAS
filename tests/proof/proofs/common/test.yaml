filenames: ["common.ads", "lmcp_messages.ads", "common_formal_containers.ads", "int64_parsing.ads"]
level: 1
