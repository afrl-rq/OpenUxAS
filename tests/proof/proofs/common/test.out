common.ads:68:14: medium: postcondition might fail, cannot prove Source (Source'First - 1 + J) = Element (To_Unbounded_String'Result, J)
common.ads:79:19: medium: postcondition might fail
common.ads:93:19: medium: postcondition might fail
common.ads:105:22: medium: postcondition might fail, cannot prove Slice'Result'First = 1
